### PR TITLE
Add Admissions Trial Files

### PR DESCRIPTION
**Pull request by renercrows**
### Admissions Trial

- Added the Admissions Trial page to a new repository.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    <div class="navbar">
+      <a class="active" href="#home">Home</a>
+      <a href="#portfolio">Portfolio Link</a>
+      <a href="#contact">Contact Link</a>
+      <a href="#about us">About Us</a>
+    </div>
+
+    <div class="title">
```
Comment (renercrows) [inline index.html:19]: Kindly add the body of the page inside a section tag

File: index.html [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+    <div class="projects">
+      <h1>Projects</h1>
+      <div class="row">
+        <img src="images/Portfolio.jpg" alt="Portfolio image 1">
+        <img src="images/Portfolio2.jpg" alt="Portfolio image 2">
+      </div>
+    </div>
+
+    <div class="footer">
```
Comment (renercrows) [inline index.html:32]: Kindly add the footer inside a section.

File: css/style.css [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+    margin-left: -8px;
+    margin-right: -8px;
+}
+
+.projects h1 {
+    margin-top: -200px;
+}
+
+.projects img {
+    width: 350px;
```
Comment (renercrows) [inline css/style.css:61]: Kindly change every width to 100% so you have a responsive webpage.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+      <a class="active" href="#home">Home</a>
+      <a href="#portfolio">Portfolio Link</a>
+      <a href="#contact">Contact Link</a>
+      <a href="#about us">About Us</a>
+    </div>
+
+    <div class="title">
+      <h1>Bilal Gujjar Portfolio</h1>
+      <h3>Web Develpment Project</h3>
+    </div>
```
Comment (renercrows) [inline index.html:22]: Kindly change the main container div tag to a header, that makes for an organized code.

File: index.html [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    <div class="navbar">
+      <a class="active" href="#home">Home</a>
+      <a href="#portfolio">Portfolio Link</a>
+      <a href="#contact">Contact Link</a>
+      <a href="#about us">About Us</a>
+    </div>
```
Comment (renercrows) [inline index.html:17]: Kindly change the main container div tag to a nav, that makes for an organized code.

File: index.html [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+      <h1>Projects</h1>
+      <div class="row">
+        <img src="images/Portfolio.jpg" alt="Portfolio image 1">
+        <img src="images/Portfolio2.jpg" alt="Portfolio image 2">
+      </div>
+    </div>
+
+    <div class="footer">
+      <h1>Footer</h1>
+    </div>
```
Comment (renercrows) [inline index.html:34]: Kindly change the main container div tag to a footer, that makes for an organized code.

File: index.html [hunk 7]
```diff
@@ -0,0 +1,50 @@
...
+      <h3>Web Develpment Project</h3>
+    </div>
+
+    <div class="projects">
+      <h1>Projects</h1>
+      <div class="row">
+        <img src="images/Portfolio.jpg" alt="Portfolio image 1">
+        <img src="images/Portfolio2.jpg" alt="Portfolio image 2">
+      </div>
+    </div>
```
Comment (renercrows) [inline index.html:30]: Kindly add the div with the class projects inside a section, that makes for an organized code.

**Review by renercrows — COMMENTED** [review]
### Changes required ♻️ 

Please check my comments below and make all required changes.

Submit for a re-review when you're done.

Thanks 😁

**Review by m05e5 — APPROVED** [review]
# Hi @renercrows , 

### Status: Approved ✔️
goodjob


:+1: Well Done.

Your project is complete! 
Congratulations! 🎉 


**You may close this Pull Request and merge the changes**

  
**Keep being awesome devs and good luck with your next milestone!**
-----

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_m05e5_**

------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**